best-practices/styling/index.html

```diff
@@ -1016,17 +1054,180 @@ <h3>Wrapping and breaking text</h3>
+⠀⠀⠠⠞⠓⠑⠀⠉⠕⠝⠉⠑⠏⠞⠀⠋⠕⠗⠀⠠⠞⠓⠑⠀⠠⠓⠊⠞⠉⠓⠓⠊⠅⠑⠗⠄⠎⠀⠀⠀
+⠠⠛⠥⠊⠙⠑⠀⠞⠕⠀⠞⠓⠑⠀⠠⠛⠁⠇⠁⠭⠽⠀⠁⠇⠇⠑⠛⠑⠙⠇⠽⠀⠉⠁⠍⠑⠀⠞⠕⠀
+⠠⠁⠙⠁⠍⠎⠀⠙⠥⠗⠊⠝⠛⠀⠁⠀⠙⠗⠥⠝⠅⠑⠝⠀⠝⠊⠛⠓⠞⠀⠊⠝⠀⠀⠀⠀⠀⠀⠀⠀
+⠠⠊⠝⠝⠎⠃⠗⠥⠉⠅⠂⠀⠠⠁⠥⠎⠞⠗⠊⠁⠲⠀⠠⠺⠓⠊⠇⠑⠀⠇⠽⠊⠝⠛⠀⠊⠝⠀⠁⠀
+⠋⠊⠑⠇⠙⠂⠀⠎⠞⠁⠗⠊⠝⠛⠀⠥⠏⠀⠁⠞⠀⠞⠓⠑⠀⠎⠞⠁⠗⠎⠂⠀⠓⠑⠀⠃⠑⠛⠁⠝
+⠏⠕⠝⠙⠑⠗⠊⠝⠛⠀⠞⠓⠑⠀⠥⠝⠊⠧⠑⠗⠎⠑⠀⠁⠝⠙⠀⠺⠓⠁⠞⠀⠅⠊⠝⠙⠀⠕⠋⠀
+⠛⠥⠊⠙⠑⠀⠎⠕⠍⠑⠕⠝⠑⠀⠺⠕⠥⠇⠙⠀⠝⠑⠑⠙⠀⠞⠕⠀⠝⠁⠧⠊⠛⠁⠞⠑⠀⠊⠞⠲
+⠠⠁⠇⠞⠓⠕⠥⠛⠓⠀⠠⠁⠙⠁⠍⠎⠀⠇⠁⠞⠑⠗⠀⠚⠕⠅⠑⠙⠀⠞⠓⠁⠞⠀⠞⠓⠑⠀⠀⠀
+⠙⠑⠞⠁⠊⠇⠎⠀⠕⠋⠀⠞⠓⠁⠞⠀⠝⠊⠛⠓⠞⠀⠺⠑⠗⠑⠀⠓⠁⠵⠽⠂⠀⠊⠞⠀⠇⠑⠙⠀
+⠞⠕⠀⠞⠓⠑⠀⠉⠗⠑⠁⠞⠊⠕⠝⠀⠕⠋⠀⠞⠓⠑⠀⠃⠑⠇⠕⠧⠑⠙⠀⠋⠊⠉⠞⠊⠕⠝⠁⠇
+⠛⠥⠊⠙⠑⠃⠕⠕⠅⠀⠞⠓⠁⠞⠀⠺⠕⠥⠇⠙⠀⠙⠑⠋⠊⠝⠑⠀⠍⠥⠉⠓⠀⠕⠋⠀⠓⠊⠎⠀
+⠉⠁⠗⠑⠑⠗⠲⠀⠠⠞⠓⠊⠎⠀⠍⠕⠍⠑⠝⠞⠀⠕⠋⠀⠎⠑⠗⠑⠝⠙⠊⠏⠊⠞⠽⠀⠊⠎⠀⠀
+⠕⠋⠞⠑⠝⠀⠉⠊⠞⠑⠙⠀⠁⠎⠀⠕⠝⠑⠀⠕⠋⠀⠞⠓⠑⠀⠍⠕⠎⠞⠀⠋⠁⠍⠕⠥⠎⠀⠀⠀
+⠦⠑⠥⠗⠑⠅⠁⠴⠀⠍⠕⠍⠑⠝⠞⠎⠀⠊⠝⠀⠇⠊⠞⠑⠗⠁⠗⠽⠀⠓⠊⠎⠞⠕⠗⠽⠲⠀⠀⠀
+⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠠⠁⠙⠁⠍⠎⠄⠀⠁⠃⠊⠇⠊⠞⠽⠀⠞⠕⠀⠃⠇⠑⠝⠙⠀⠞⠓⠑⠀⠍⠥⠝⠙⠁⠝⠑⠀⠀
+⠺⠊⠞⠓⠀⠞⠓⠑⠀⠉⠕⠎⠍⠊⠉⠀⠺⠁⠎⠀⠁⠀⠎⠊⠛⠝⠁⠞⠥⠗⠑⠀⠞⠗⠁⠊⠞⠀⠕⠋
+⠓⠊⠎⠀⠺⠗⠊⠞⠊⠝⠛⠲⠀⠠⠺⠓⠑⠞⠓⠑⠗⠀⠊⠞⠀⠺⠁⠎⠀⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠃⠥⠗⠑⠁⠥⠉⠗⠁⠞⠊⠉⠀⠁⠇⠊⠑⠝⠀⠗⠁⠉⠑⠀⠞⠗⠽⠊⠝⠛⠀⠞⠕⠀⠀⠀⠀⠀⠀⠀
+⠙⠑⠍⠕⠇⠊⠎⠓⠀⠠⠑⠁⠗⠞⠓⠀⠞⠕⠀⠍⠁⠅⠑⠀⠺⠁⠽⠀⠋⠕⠗⠀⠁⠝⠀⠀⠀⠀⠀⠀
+⠊⠝⠞⠑⠗⠎⠞⠑⠇⠇⠁⠗⠀⠓⠊⠛⠓⠺⠁⠽⠀⠕⠗⠀⠁⠀⠙⠑⠏⠗⠑⠎⠎⠑⠙⠀⠀⠀⠀⠀</code></pre>
+				</aside>				
+				<div class="issue" data-number="93" title="Need HTML and CSS of avoiding page breaks between
 				two elements"><span/></div>
-
-				<div class="issue" data-number="93" title="Seeking an example of avoiding orphan or widow
+				<aside class="example" title="THE END kept on last line page so it stays with other material (rather than with a blank before)"></aside>
+				<p>In order to obtain the following desired layout for the above HTML code,</p>
+				<pre><code class="formatted-braille">⠎⠓⠑⠑⠏⠤⠃⠑⠇⠇⠎⠂⠀⠁⠝⠙⠀⠞⠓⠑⠀⠠⠟⠥⠑⠑⠝⠄⠎⠀⠎⠓⠗⠊⠇⠇⠀⠀⠀⠀
+⠉⠗⠊⠑⠎⠀⠞⠕⠀⠞⠓⠑⠀⠧⠕⠊⠉⠑⠀⠕⠋⠀⠞⠓⠑⠀⠎⠓⠑⠏⠓⠑⠗⠙⠀⠀⠀⠀⠀⠀
+⠃⠕⠽⠤⠤⠁⠝⠙⠀⠞⠓⠑⠀⠎⠝⠑⠑⠵⠑⠀⠕⠋⠀⠞⠓⠑⠀⠃⠁⠃⠽⠂⠀⠞⠓⠑⠀⠀⠀⠀
+⠎⠓⠗⠊⠑⠅⠀⠕⠋⠀⠞⠓⠑⠀⠠⠛⠗⠽⠏⠓⠕⠝⠂⠀⠁⠝⠙⠀⠁⠇⠇⠀⠞⠓⠑⠀⠀⠀⠀⠀
+⠕⠞⠓⠑⠗⠀⠟⠥⠑⠑⠗⠀⠝⠕⠊⠎⠑⠎⠂⠀⠺⠕⠥⠇⠙⠀⠉⠓⠁⠝⠛⠑⠀⠐⠣⠎⠓⠑⠀⠀
+⠅⠝⠑⠺⠐⠜⠀⠞⠕⠀⠞⠓⠑⠀⠉⠕⠝⠋⠥⠎⠑⠙⠀⠉⠇⠁⠍⠕⠥⠗⠀⠕⠋⠀⠞⠓⠑⠀⠀⠀
+⠃⠥⠎⠽⠀⠋⠁⠗⠍⠤⠽⠁⠗⠙⠤⠤⠺⠓⠊⠇⠑⠀⠞⠓⠑⠀⠇⠕⠺⠊⠝⠛⠀⠕⠋⠀⠞⠓⠑⠀
+⠉⠁⠞⠞⠇⠑⠀⠊⠝⠀⠞⠓⠑⠀⠙⠊⠎⠞⠁⠝⠉⠑⠀⠺⠕⠥⠇⠙⠀⠞⠁⠅⠑⠀⠞⠓⠑⠀⠀⠀
+⠏⠇⠁⠉⠑⠀⠕⠋⠀⠞⠓⠑⠀⠠⠍⠕⠉⠅⠀⠠⠞⠥⠗⠞⠇⠑⠄⠎⠀⠓⠑⠁⠧⠽⠀⠎⠕⠃⠎⠲
+⠀⠀⠠⠇⠁⠎⠞⠇⠽⠂⠀⠎⠓⠑⠀⠏⠊⠉⠞⠥⠗⠑⠙⠀⠞⠕⠀⠓⠑⠗⠎⠑⠇⠋⠀⠓⠕⠺⠀⠀
+⠞⠓⠊⠎⠀⠎⠁⠍⠑⠀⠇⠊⠞⠞⠇⠑⠀⠎⠊⠎⠞⠑⠗⠀⠕⠋⠀⠓⠑⠗⠎⠀⠺⠕⠥⠇⠙⠂⠀⠀
+⠊⠝⠀⠞⠓⠑⠀⠁⠋⠞⠑⠗⠤⠞⠊⠍⠑⠂⠀⠃⠑⠀⠓⠑⠗⠎⠑⠇⠋⠀⠁⠀⠛⠗⠕⠺⠝⠀⠀⠀
+⠺⠕⠍⠁⠝⠆⠀⠁⠝⠙⠀⠓⠕⠺⠀⠎⠓⠑⠀⠺⠕⠥⠇⠙⠀⠅⠑⠑⠏⠂⠀⠞⠓⠗⠕⠥⠛⠓⠀⠀
+⠁⠇⠇⠀⠓⠑⠗⠀⠗⠊⠏⠑⠗⠀⠽⠑⠁⠗⠎⠂⠀⠞⠓⠑⠀⠎⠊⠍⠏⠇⠑⠀⠁⠝⠙⠀⠀⠀⠀⠀
+⠇⠕⠧⠊⠝⠛⠀⠓⠑⠁⠗⠞⠀⠕⠋⠀⠓⠑⠗⠀⠉⠓⠊⠇⠙⠓⠕⠕⠙⠒⠀⠁⠝⠙⠀⠓⠕⠺⠀⠀	
+⠎⠓⠑⠀⠺⠕⠥⠇⠙⠀⠛⠁⠞⠓⠑⠗⠀⠁⠃⠕⠥⠞⠀⠓⠑⠗⠀⠕⠞⠓⠑⠗⠀⠇⠊⠞⠞⠇⠑⠀
+⠉⠓⠊⠇⠙⠗⠑⠝⠂⠀⠁⠝⠙⠀⠍⠁⠅⠑⠀⠠⠠⠞⠓⠑⠊⠗⠀⠑⠽⠑⠎⠀⠃⠗⠊⠛⠓⠞⠀⠀
+⠁⠝⠙⠀⠑⠁⠛⠑⠗⠀⠺⠊⠞⠓⠀⠍⠁⠝⠽⠀⠁⠀⠎⠞⠗⠁⠝⠛⠑⠀⠞⠁⠇⠑⠂⠀⠀⠀⠀⠀
+⠏⠑⠗⠓⠁⠏⠎⠀⠑⠧⠑⠝⠀⠺⠊⠞⠓⠀⠞⠓⠑⠀⠙⠗⠑⠁⠍⠀⠕⠋⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠠⠺⠕⠝⠙⠑⠗⠇⠁⠝⠙⠀⠕⠋⠀⠇⠕⠝⠛⠀⠁⠛⠕⠒⠀⠁⠝⠙⠀⠓⠕⠺⠀⠎⠓⠑⠀⠀⠀⠀
+⠺⠕⠥⠇⠙⠀⠋⠑⠑⠇⠀⠺⠊⠞⠓⠀⠁⠇⠇⠀⠞⠓⠑⠊⠗⠀⠎⠊⠍⠏⠇⠑⠀⠀⠀⠀⠀⠀⠀⠀
+⠎⠕⠗⠗⠕⠺⠎⠂⠀⠁⠝⠙⠀⠋⠊⠝⠙⠀⠁⠀⠏⠇⠑⠁⠎⠥⠗⠑⠀⠊⠝⠀⠁⠇⠇⠀⠀⠀⠀⠀
+⠞⠓⠑⠊⠗⠀⠎⠊⠍⠏⠇⠑⠀⠚⠕⠽⠎⠂⠀⠗⠑⠍⠑⠍⠃⠑⠗⠊⠝⠛⠀⠓⠑⠗⠀⠕⠺⠝⠀⠀
+⠉⠓⠊⠇⠙⠤⠇⠊⠋⠑⠂⠀⠁⠝⠙⠀⠞⠓⠑⠀⠓⠁⠏⠏⠽⠀⠎⠥⠍⠍⠑⠗⠀⠙⠁⠽⠎⠲⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠨⠣⠠⠞⠓⠑⠀⠠⠑⠝⠙⠈⠨⠜⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀</code></pre>
+				</aside>
+				<div class="issue" data-number="93" title="Need HTML and CSS of avoiding orphan or widow
 				lines"><span/></div>
-
+				<aside class="example" title="Heading 2 moving to next page to remain with material"></aside>
+					<p>In order to obtain the following desired layout for the above HTML code,</p>
+					<pre><code class="formatted-braille">⠠⠔⠞⠗⠕⠙⠥⠉⠰⠝⠀⠞⠕⠀⠠⠡⠑⠑⠎⠑
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠠⠡⠑⠑⠎⠑⠀⠊⠎⠀⠐⠕⠀⠷⠀⠮⠀⠕⠇⠙⠑⠌⠀⠯⠀⠍⠕⠌⠀⠧⠻⠎⠁⠞⠊⠇⠑⠀
```

Comment:
Like in the "Forced page break between front matter and body matter" example, there are a lot of things going on in this example as well: different types of headings, paragraphs and list items. Would benefit from some simplification (to better focus on what we're trying to show).

Reply:
Would you feel comfortable just deleting any aspects that seem unnecessary or are you asking me to go in and edit it? The only things you need are the last line before the end of the braille page, the heading itself, and then the line after the heading itself. The braille page break could be indicated in some other way but in the current example, it is done using a braille page number. If you have another suggestion here, I think that would work.

Reply:
No, not asking you to go in and actually edit it. Just wanted to know what you think before I would go mess with the example.

Yes, I think it would be good to simplify/condense the example a bit. I think I can figure it out.

Reply:
By the way another issue with example is that it is in contracted braille. I think all other examples are uncontracted (and it's also what we requested). But we can fix that another time.